front-end/src/components/Post/Poll/CouncilSignals.tsx

```diff
@@ -88,7 +89,7 @@ const CouncilSignals = ({ className, data }: Props) => {
 
 	return (
 		<Card className={className}>
-			<h3>Council Signals</h3>
+			<h3>Council Signals <HelperTooltip content='This represent the offchain votes of council members' /></h3>
```

Comment:
typo I tend to write offchain all together but I think it should probably be with a - or a space.
```suggestion
			<h3>Council Signals <HelperTooltip content='This represents the off-chain votes of council members/></h3>
```